app/components/Search/index.js

```diff
@@ -53,12 +54,14 @@ const Wrapper = styled.div`
   width: 100%;
   z-index: 1;
   position: relative;
+  margin-bottom: ${({ active }) => (active ? '40px' : '0px')};
```

Comment:
Or 
```jsx
${({ active }) => active && css`margin-bottom: 40px;`};
```

and at the top:
```jsx
import styled, { css } from 'styled-components';
```

Reply:
I'll leave it for now. You're suggestion would definitely be a cleaner solution if there are more properties that need to be changed when active.